pages/documentation/migration-V3.md

```diff
@@ -0,0 +1,833 @@
+            &lt;img src="{{ your uswds image path }}/usa-icons-bg/search--white.svg" 
+              class="usa-search__submit-icon" alt="Search"&gt;
+          &lt;/button&gt;
+          </code>
+        </pre>
+      </li>
+    </ol>   
+    <p><strong>[MARKUP] Update social media icons in the footer.</strong></p>
+    <p>
+     You'll need to update social media icons in the USWDS footer. We're now using explicit images to better support legibility in instances where icons do not load. 
+    </p>
+    <strong>What to do:</strong>
+    <ol>
+      <li> Check your codebase for instances of <strong>usa-social-link.</strong></li>
+      <li> Update the markup from the old version to the new version if you use it.
+        <p><strong>Old</strong></p>
+        <pre>
+          <code>
+            &lt;a class="usa-social-link usa-social-link--facebook" href="{{ link }}"&gt;
+              &lt;span&gt;Facebook&lt;/span&gt;
+            &lt;/a&gt;
+            &lt;a class="usa-social-link usa-social-link--twitter" href="{{ link }}"&gt;
+              &lt;span&gt;Twitter&lt;/span&gt;
+            &lt;/a&gt;
+            &lt;a class="usa-social-link usa-social-link--youtube" href="{{ link }}"&gt;
+              &lt;span&gt;YouTube&lt;/span&gt;
+            &lt;/a&gt;
+            &lt;a class="usa-social-link usa-social-link--instagram" href="{{ link }}"&gt;
+              &lt;span&gt;Instagram&lt;/span&gt;
+            &lt;/a&gt;
+            &lt;a class="usa-social-link usa-social-link--rss" href="{{ link }}"&gt;
+              &lt;span&gt;RSS&lt;/span&gt;
+            &lt;/a&gt;
+          </code>
+        </pre>
+        <p><strong>New</strong></p>
+        <pre>
+          <code>
+           &lt;a class="usa-social-link" href="{{ link }}"&gt;
+            &lt;img class="usa-social-link__icon" 
+              src="{{ your uswds image path }}/usa-icons/facebook.svg" 
+              alt="Facebook"&gt;
+          &lt;/a&gt;
+          &lt;a class="usa-social-link" href="{{ link }}"&gt;
+            &lt;img class="usa-social-link__icon" 
+              src="{{ your uswds image path }}/usa-icons/twitter.svg" 
+              alt="Twitter"&gt;
+          &lt;/a&gt;
+          &lt;a class="usa-social-link" href="{{ link }}"&gt;
+            &lt;img class="usa-social-link__icon" 
+              src="{{ your uswds image path }}/usa-icons/youtube.svg" 
+              alt="YouTube"&gt;
+          &lt;/a&gt;
+          &lt;a class="usa-social-link" href="{{ link }}"&gt;
+            &lt;img class="usa-social-link__icon" 
+              src="{{ your uswds image path }}/usa-icons/instagram.svg" 
+              alt="Instagram"&gt;
+          &lt;/a&gt;
+          &lt;a class="usa-social-link" href="{{ link }}"&gt;
+            &lt;img class="usa-social-link__icon" 
+              src="{{ your uswds image path }}/usa-icons/rss_feed.svg" 
+              alt="RSS"&gt;
+          &lt;/a&gt;
+          </code>
+        </pre>
```

Comment:
@mahoneycm  To clear out the liquid syntax errors, we need to wrap the code samples that have double braces `{{` with `{% raw %}`/`{% endraw %}`

Additionally, when working with `<code`/`<pre>`, it will preserve the tab/spacing in the code, so we need to reformat our code inside `<pre>` to remove extra tabbing and line breaks. 

```suggestion
               <pre>
          <code>{% raw %}
&lt;a class="usa-social-link" href="{{ link }}"&gt;
  &lt;img class="usa-social-link__icon" 
    src="{{ your uswds image path }}/usa-icons/facebook.svg" 
    alt="Facebook"&gt;
&lt;/a&gt;
&lt;a class="usa-social-link" href="{{ link }}"&gt;
  &lt;img class="usa-social-link__icon"
    src="{{ your uswds image path }}/usa-icons/twitter.svg" 
    alt="Twitter"&gt;
&lt;/a&gt;
&lt;a class="usa-social-link" href="{{ link }}"&gt;
  &lt;img class="usa-social-link__icon" 
    src="{{ your uswds image path }}/usa-icons/youtube.svg" 
    alt="YouTube"&gt;
&lt;/a&gt;
&lt;a class="usa-social-link" href="{{ link }}"&gt;
  &lt;img class="usa-social-link__icon" 
    src="{{ your uswds image path }}/usa-icons/instagram.svg" 
    alt="Instagram"&gt;
&lt;/a&gt;
&lt;a class="usa-social-link" href="{{ link }}"&gt;
  &lt;img class="usa-social-link__icon" 
    src="{{ your uswds image path }}/usa-icons/rss_feed.svg" 
    alt="RSS"&gt;
&lt;/a&gt;{% endraw %}</code>
        </pre>
```

![image](https://user-images.githubusercontent.com/93996430/164060219-f465c174-6f01-4a5e-ae44-1e4eda8bf309.png)

Reply:
Completed revision in [0d53afb](https://github.com/uswds/uswds-site/pull/1529/commits/0d53afb451926ca992073b38fae0dbecff00fba9)